ci/jenkins-track/scalajs-task-worker.sh

```diff
@@ -0,0 +1,67 @@
+# Launcher script for task worker (dispatched by matrix-build)
+
+# Setup sbt home dirs
+
+LOC_SBT_BASE=/localhome/jenkins/scala-js-sbt-homes
+LOC_SBT_BOOT=$LOC_SBT_BASE/sbt-boot
+LOC_SBT_HOME=$LOC_SBT_BASE/sbt-home
+
+mkdir -p $LOC_SBT_BOOT
+mkdir -p $LOC_SBT_HOME
+
+### Create tmp.sh ###
+
+# Node module path
+
+echo 'export NODE_PATH="$HOME/node_modules/"' > tmp.sh
+
+# sbt options
+
+echo 'SBT_OPTS="' \
+    "-J-Xmx3G -J-XX:MaxPermSize=512M" \
+    "-Djline.terminal=jline.UnsupportedTerminal" \
+    "-Dsbt.boot.directory=$LOC_SBT_BOOT" \
+    "-Dsbt.ivy.home=$LOC_SBT_HOME" \
+    "-Divy.home=$LOC_SBT_HOME" \
+    "-Dsbt.global.base=$LOC_SBT_BASE" \
+    '"' >> tmp.sh
+
+# Define setJavaVersion
+
+echo 'setJavaVersion() {' \
+     '  export JAVA_HOME=$HOME/apps/java-$1' \
+     '}' >> tmp.sh
+
+# Define sbt and sbtretry
+
+echo 'sbtretry() {' \
+     '  local TIMEOUT=25m' \
+     '  echo "RUNNING timeout -k 5 $TIMEOUT sbt" "$@"' \
```

Comment:
Argh... Stupid me... And with the `'` quotes I cannot write `\n`... And if I use `"` quotes, stuff gets interpolated... :(

Why is it I cry every time I have to deal with Jenkins?